commands/start-container.ts

```diff
@@ -19,4 +19,11 @@ export function startContainer() {
 
 export function startContainerInteractive() {
     doStartContainer(true);
+}
+
+export function startAzureCLI() {
+    let terminal: vscode.Terminal = vscode.window.createTerminal('Azure CLI');
+    terminal.sendText(`docker run -it --rm azure-xplat-cli:latest`);
```

Comment:
Do we want to use the `microsoft/azure-cli` image? I tried running this command as-is and the `azure-xplat-cli` couldn't be found.